app/src/forms/rbac/routes.js

```diff
@@ -15,39 +14,39 @@ routes.get('/idps', async (req, res, next) => {
   await controller.getIdentityProviders(req, res, next);
 });
 
-routes.get('/forms', keycloak.protect(`${config.get('server.keycloak.clientId')}:admin`), async (req, res, next) => {
+routes.get('/forms', async (req, res, next) => {
   await controller.getFormUsers(req, res, next);
 });
 
-routes.put('/forms', keycloak.protect(`${config.get('server.keycloak.clientId')}:admin`), async (req, res, next) => {
+routes.put('/forms', async (req, res, next) => {
   await controller.setFormUsers(req, res, next);
 });
 
-routes.get('/users', keycloak.protect(`${config.get('server.keycloak.clientId')}:admin`), async (req, res, next) => {
+routes.get('/users', async (req, res, next) => {
   await controller.getUserForms(req, res, next);
 });
 
-routes.put('/users', keycloak.protect(`${config.get('server.keycloak.clientId')}:admin`), async (req, res, next) => {
+routes.put('/users', async (req, res, next) => {
   await controller.setUserForms(req, res, next);
 });
 
-routes.get('/', keycloak.protect(`${config.get('server.keycloak.clientId')}:admin`), async (req, res, next) => {
+routes.get('/', async (req, res, next) => {
```

Comment:
I don't think these functions/endpoints (list, create, read, update, delete) are in use on the UX, I would either remove them, deprecate them (410?) or leave the admin check.  Don't want these "wide" open.  Admin check left in at a minimum - maybe moved under /admin/rbac (would also apply to roles  & permissions)?

Reply:
Yeah we should take them out I think until needed if ever.

Reply:
Yeah will leave downstream methods but chop the route for now.